package.json

```diff
@@ -27,7 +27,7 @@
     "lint:css": "stylelint ./**/*.less",
     "lint-fix": "npm run lint-fix:js && npm run lint-fix:css",
     "lint-fix:js": "eslint --fix --ext js,vue .",
-    "lint-fix:css": "stylelint --fix ./**/*.{less,vue}",
+    "lint-fix:css": "stylelint --fix ./**/*.less",
```

Comment:
We are already using eslint for vue files as can be seen in the line above.

Reply:
Ah, vue files also contain CSS in their <style> tags; I think that's why we were applying it to both?

Reply:
I figured out how to get vue working with stylelint but it looks like it wasn't enabled because many many changes
![image](https://github.com/user-attachments/assets/f72363f9-82ec-4d89-b28c-2d64195e09af)